src/Compilers/Core/Portable/Emit/EditAndContinue/EmitBaseline.cs

```diff
@@ -59,15 +59,18 @@ public sealed class EmitBaseline
         internal sealed class MetadataSymbols
         {
             public readonly IReadOnlyDictionary<AnonymousTypeKey, AnonymousTypeValue> AnonymousTypes;
+            public readonly ImmutableDictionary<AssemblyIdentity, AssemblyIdentity> AssemblyReferenceIdentityMap;
```

Comment:
Please add a comment describing what values are included and what values aren't included into this map.